tests/api/Jalasoft.TeamUp.Resumes.API.Tests/PostResumesApiTests.cs

```diff
@@ -33,5 +34,15 @@ public void PostResume_Returns_CreateResume_Resume()
             var createdResult = Assert.IsType<CreatedResult>(response);
             Assert.IsType<Resume>(createdResult.Value);
         }
+
+        [Fact]
+        public void PostResume_Returns_BadRequest()
```

Comment:
follow the convention we are using "MethodName_Context_Result"
should be "PostResume_InvalidResume_BadRequest"

Reply:
done